backend/src/Request/Admin/AdminProfileUpdateRequest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Request\Admin;
+
+class AdminProfileUpdateRequest
```

Comment:
no need for this class at all, you can use the previous one, just rename it to be "AdminProfileRequest"